[2023-09-19T11:31:11.635+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: udac_example_dag.stage_events_to_redshift_task manual__2023-09-19T11:31:08.853360+00:00 [queued]>
[2023-09-19T11:31:11.640+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: udac_example_dag.stage_events_to_redshift_task manual__2023-09-19T11:31:08.853360+00:00 [queued]>
[2023-09-19T11:31:11.640+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-09-19T11:31:11.656+0000] {taskinstance.py:1382} INFO - Executing <Task(StageToRedshiftOperator): stage_events_to_redshift_task> on 2023-09-19 11:31:08.853360+00:00
[2023-09-19T11:31:11.660+0000] {standard_task_runner.py:57} INFO - Started process 2007 to run task
[2023-09-19T11:31:11.662+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'udac_example_dag', 'stage_events_to_redshift_task', 'manual__2023-09-19T11:31:08.853360+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/udac_example_dag.py', '--cfg-path', '/tmp/tmpk1ushbq5']
[2023-09-19T11:31:11.664+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask stage_events_to_redshift_task
[2023-09-19T11:31:11.699+0000] {task_command.py:415} INFO - Running <TaskInstance: udac_example_dag.stage_events_to_redshift_task manual__2023-09-19T11:31:08.853360+00:00 [running]> on host cd9223bba8c3
[2023-09-19T11:31:11.748+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kstine' AIRFLOW_CTX_DAG_ID='udac_example_dag' AIRFLOW_CTX_TASK_ID='stage_events_to_redshift_task' AIRFLOW_CTX_EXECUTION_DATE='2023-09-19T11:31:08.853360+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-19T11:31:08.853360+00:00'
[2023-09-19T11:31:11.757+0000] {base.py:73} INFO - Using connection ID 'redshift_connection' for task execution.
[2023-09-19T11:31:13.332+0000] {sql.py:418} INFO - Running statement: 
        COPY staging_events
        FROM 's3://kstine-***-pipeline/project/log-data/'
        ACCESS_KEY_ID 'AKIAX5RTZI43UEOGKP4P'
        SECRET_ACCESS_KEY '***'
        IGNOREHEADER 1
        DELIMITER ','
    , parameters: None
[2023-09-19T11:31:14.177+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/plugins/operators/stage_redshift.py", line 47, in execute
    response = hook.run(query)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 392, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 423, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.InternalError_: Load into table 'staging_events' failed.  Check 'sys_load_error_detail' system table for details.

[2023-09-19T11:31:14.183+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=udac_example_dag, task_id=stage_events_to_redshift_task, execution_date=20230919T113108, start_date=20230919T113111, end_date=20230919T113114
[2023-09-19T11:31:14.193+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 20 for task stage_events_to_redshift_task (Load into table 'staging_events' failed.  Check 'sys_load_error_detail' system table for details.
; 2007)
[2023-09-19T11:31:14.202+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-19T11:31:14.219+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
